apps/scan/backend/src/scanners/custom/app_scan.test.ts

```diff
@@ -88,10 +92,8 @@ test('configure and scan hmpb', async () => {
         type: 'ValidSheet',
       };
 
-      mockScanner.scan.mockResolvedValue(ok(await ballotImages.completeHmpb()));
+      simulateScan(mockScanner, await ballotImages.completeHmpb());
       await apiClient.scanBallot();
-      await expectStatus(apiClient, { state: 'scanning' });
```

Comment:
With this new approach, we can't guarantee that the state machine will stay in the `scanning` state long enough to check it reliably, so I had to just remove that check, which is not ideal. The only reason it was reliably in the `scanning` state before is that it would be stuck in a retry loop.